pkg/flows/config/blueprints/otto.yaml

```diff
@@ -0,0 +1,20 @@
+flows:
+  otto:
+    default: true
+    globals:
+      ingestion:
+        textsplitter:
```

Comment:
Does this mean you are just using textsplitter until you write the "enhanced" splitter.

Reply:
Yes - but the new one will land today